src/main/java/seedu/address/logic/commands/ScheduleCommand.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.candidate.Candidate;
+import seedu.address.model.interview.Interview;
+
+/**
+ * Schedules a candidate identified using it's displayed index from the address book for an interview
+ * on a specified time slot.
+ */
+public class ScheduleCommand extends Command {
+
+    public static final String COMMAND_WORD = "schedule";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Schedules the candidate identified by the index number for an interview on given date and time.\n"
+            + "Time and date given must not be in the past.\n"
```

Comment:
Just nitpicking here, but maybe the `Time and date` could be in the same sequence, `Date and time`, as the other string messages.

Reply:
Good catch En En! Will fix it in subsequent commits